driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -784,6 +803,11 @@ public Builder withMaxSchemaAgreementWaitSeconds(int maxSchemaAgreementWaitSecon
          * (the default), then the native protocol version 1 will be use for the lifetime
          * of the Cluster instance.
          * <p/>
+         * With {@link ProtocolOptions#setAllowBetaProtocolVersions(boolean)}, it is
```

Comment:
If the referenced method is deleted as I suggested, please delete this paragraph as well (or rephrase accordingly).